src/fugw/solvers/utils.py

```diff
@@ -938,18 +1077,96 @@ def elementwise_prod_fact_sparse(a, b, p):
     )
 
 
+def compute_approx_kl(p, q):
```

Comment:
Not really clear what approx_kl is supposed to be (ref ?)

Again, for publica functions add at elast 1-line docstring.

Reply:
Indeed, this naming is misleading.
The idea is that what we are computing here is not exactly a KL divergence, because p and q are not necessarily probability measures.
@6Ulm and I suggest we replace `_approx_` with `unnormalized` or `generalized`. What do you think?

Reply:
+1 for unnormalized

Reply:
Yaaay, unnormalized it is!